### ISSUE TITLE
Dash to dock can't be rearranged.

### ISSUE THREAD

**Issue by thedarkraze** [Maestroschan/appfolders-manager-gnome-extension#83]
As the title says. While the extension is turned on, the drag and drop rearranging no longer functions properly for the Dash to Dock extension. Using gnome 3.32.1.

**Comment by maoschanz** [comment]
What version of the extension do you use? (it's written in the preferences dialog)

**Comment by thedarkraze** [comment]
v16

**Comment by Natetronn** [comment]
I'm hitting this as well. The following is what led me here:

https://github.com/micheleg/dash-to-dock/issues/959#issuecomment-550469545

I can confirm the work around worked. I'm on v16.

**Comment by maoschanz** [comment]
yes i'll fix it when i'll have the time to downgrade to these old versions

**Comment by maoschanz** [comment]
(alternatively, you can get the trace from `journalctl -f /usr/bin/gnome-shell` and post it here)

**Comment by Natetronn** [comment]
> (alternatively, you can get the trace from `journalctl -f /usr/bin/gnome-shell` and post it here)

Is there anything I need to do other than run the command? I only see what appears to be a Firefox and a Guake warning.

**Comment by maoschanz** [comment]
it's the live stream of the logs on the system, if you reproduce the bug while it's running you should get something related to one of the extensions

**Comment by Natetronn** [comment]
Not getting anything in the log related to either extension when reproducing the bug. Turned off, I can drag and drop sort dock icons. When turned on, I can drag but, there is no movement from the other icons moving out of the way or highlight happening in the spot the drop is about to take place in. When I drop the icon just returns to the original location.

**Comment by Natetronn** [comment]
Maybe this:

> Nov 21 12:14:51 username gnome-shell[1216]: Some code accessed the property 'discreteGpuAvailable' on the module 'appDisplay'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.

> Nov 21 12:14:51 username gnome-shell[1216]: Some code accessed the property 'WindowPreviewMenuItem' on the module 'windowPreview'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.

**Comment by maoschanz** [comment]
i think these are warnings for dash-to-dock

Anyway, if it's such a mysterious bug you'll really have to wait until i install some ubuntu 18.04 somewhere. Meanwhile, as stated in the prefs/about dialog of the extension, it's totally fine to disable it, it will not undo the folders or anything else, enable it only when you need it
